docs/framework_config.rst

```diff
@@ -304,7 +305,7 @@ the built-in Economy credits::
 
     class Pets(commands.Cog):
         def __init__(self):
-            self.config = Config.get_conf(self, 1234567890)
+            self.config = Config.get_conf(self, 1234567890, force_registration=True)
```

Comment:
Should this use `identifier` as a kwarg, to match other examples?

Reply:
Variety in examples shows that there are multiple valid ways of using the method - seems fine to me.